src/components/chapterList/styles.js

```diff
@@ -0,0 +1,110 @@
+import { makeStyles } from '@material-ui/core'
+import { borderRadius, width } from '@mui/system'
+import { colors } from '../../styles/colors'
+
+const useStyles = makeStyles({
+  mainbox: {
+    width: '77%',
+    height: '80%',
+    padding: '8%',
+    paddingTop: '3%',
+    color: colors.blue,
+    marginLeft: '15%',
+    position: 'absolute'
+  },
+
+  heading: {
+    fontSize: '36px',
+    lineHeight: '96px',
+    fontFamily: 'Source Sans Pro',
+
+    width: '62%', //586px
+    height: '5%px' //96px
```

Comment:
Do we need manual height and width in Typography also? Check after removing them